presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -364,44 +364,44 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .aggregate(CovarianceAggregation.class)
                 .aggregate(RegressionAggregation.class)
                 .aggregate(CorrelationAggregation.class)
-                .scalar(SequenceFunction.class)
```

Comment:
Let's leave the name "scalar" to be consistent with the other methods. Also, to minimize unnecessary changes.